XML/TestCases/PerformanceTests.xml

```diff
@@ -124,7 +124,7 @@
         <Category>Performance</Category>
         <Area>NETWORK</Area>
         <Tags>network,hv_netvsc</Tags>
-        <Priority>1</Priority>
+        <Priority></Priority>
```

Comment:
missed last 4?

Reply:
no, we are not recommended to run those cases, like what we have in Other.xml